doc/tutorials.rst

```diff
@@ -20,37 +60,119 @@ plugins, and devices, via simple and easy-to-follow examples.
 
 .. customgalleryitem::
     :tooltip: Use quantum machine learning to tune a beamsplitter.
-    :figure: ../examples/figures/gaussian_transformation.png
+    :figure: ../examples/figures/gaussiancircuit.png
     :description: :ref:`gaussian_transformation`
 
+:html:`<div style='clear:both'></div>`
+
+Classical interfaces
+~~~~~~~~~~~~~~~~~~~~~
+
+The following tutorials take an in-depth look at one and the same Ising model tutorial, implemented across different interfaces offered by PennyLane. 
+
+.. toctree::
+    :hidden:
+    :maxdepth: 2
+
+    tutorials/pennylane_isingmodel_NumPy 
+    tutorials/pennylane_isingmodel_TF
+    tutorials/pennylane_isingmodel_PyTorch
+
+.. customgalleryitem::
+    :tooltip: Ising model example with PennyLane NumPy interface.
+    :figure: ../examples/figures/pennylane_xanadu.png
+    :description: :ref:`isingmodel_NumPy`
+
+.. customgalleryitem::
+    :tooltip: Ising model example with PennyLane TensorFlow interface.
+    :figure: ../examples/figures/tensorflow.png
+    :description: :ref:`isingmodel_TF`
+
+.. customgalleryitem::
+    :tooltip: Ising model example with PennyLane PyTorch interface.
+    :figure: ../examples/figures/pytorch.png
+    :description: :ref:`isingmodel_PyTorch`
+
+:html:`<div style='clear:both'></div>`
+
+
+Plugins and hybrid computation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The most powerful design feature of PennyLane is that it allows the user to run hybrid computations on a variety of different simulators and real quantum hardware. The following tutorials introduce the notion of hybrid computation by combining several PennyLane plugins.
+
+
+.. toctree::
+    :hidden:
+    :maxdepth: 1
+
+    tutorials/pennylane_plugins_hybrid
+    tutorials/pennylane_pytorch_noise
+    tutorials/pennylane_run_state_preparation
```

Comment:
I'm getting a 

TypeError: Can't instantiate abstract class QVMDevice with abstract methods observables

when building the docs. The error shows in the tutorial after the line "dev = qml.device("forest.qvm", device="3q-pyqvm", shots=1000)"

Reply:
@mariaschuld did this issue workout?